docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -76,302 +82,44 @@ Installing YugabyteDB Anywhere on Airgapped hosts, without access to any Interne
 
 ### Kubernetes-based installations
 
-For a [Kubernetes-based installation](#kubernetes-based-installations), you need to ensure that the host can pull container images from the [Quay.io](https://quay.io/) container registry. For details, see [Pull and push YugabyteDB Docker images to private container registry](#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
+The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+
+- Kubernetes 1.22
+- Helm 3.10
 
-In addition, you need to ensure that core dumps are enabled and configured on the underlying Kubernetes node. For details, see [Specify ulimit and remember the location of core dumps](#specify-ulimit-and-remember-the-location-of-core-dumps). 
+
+Before installing YugabyteDB Anywhere, verify that you have the following:
+
+- A Kubernetes cluster configured with [Helm](https://helm.sh/).
```

Comment:
minor: this statement sounds technically incorrect. helm is a command line tool installed on the client, whereas the cluster is server side. Maybe we could say 

1. A Kubernetes cluster with nodes configured according to the hardware requirements above.
2. A client environment with the kubectl and helm command line tools configured with a service account or user that has admin access to a single namespace on the above cluster.
3. A kubernetes secret

Reply:
Changed